techniques/general/G64.html

```diff
@@ -2,143 +2,113 @@
-            <p>This is one of a series of techniques for locating content that are sufficient for
-               addressing Success Criterion 2.4.5. 
-               A table of contents provides links to sections and subsections of the same document.
-               The information in the document is usually organized hierarchically, and is intended
-               to be read sequentially. Just as there could be many books in a library, each with
-               its own table of contents, a Web site may contain many documents, each with its own
-               table of contents.
+            <h2>解説</h2>
+            <p>これは、達成基準 2.4.5 の実装に十分なコンテンツを配置するための一連の達成方法の一つである。目次は、同じドキュメントのセクションとサブセクションへのリンクを提供する。ドキュメントにある情報は、通常、階層的に構成され、順を追って読まれることを意図している。まさに図書館にある多くの本にそれぞれ目次があるように、ウェブサイトの多くのドキュメントにも目次がある。
             </p>
-            <p>The table of contents serves two purposes:</p>
+            <p>目次は次の二つの目的がある:</p>
             <ul>
                
-               <li>It gives users an overview of the document's contents and organization.</li>
-               
-               <li>It allows readers to go directly to a specific section of an on-line document.</li>
+               <li>ドキュメントの内容及び構成の概要を利用者に与える。</li>
                
+               <li>読者がオンラインドキュメントの特定のセクションに直接行けるようにする。</li>
+            
             </ul>
-            <p>The table of contents typically includes only major sections of the document, though
-               in some cases an expanded table of contents that provides a more detailed view of
-               a complex document may be desirable.
+            <p>目次はドキュメントの主要なセクションだけを含むことが多いが、場合によっては複雑なドキュメントのより詳細な一覧を提供する展開された目次が望ましいかもしれない。
             </p>
-            <p>The sections of the document could be located on the same Web page or divided into
-               multiple Web pages. A table of contents is particularly useful when a document is
-               divided into multiple Web pages.
+            <p>ドキュメントのセクションは、同じウェブページに配置する、又は複数のウェブページに分割して配置することができる。目次は、ドキュメントが複数のウェブページに分割されるとき、特に役に立つ。
             </p>
-            <p>There is a distinction between a table of contents and other Navigational elements
-               such as a Navigation Bar or Site Map. A table of contents provides links to sections
-               of the same document. Those sections could be located on the same Web page or spread
-               across multiple Web pages. But together, they make a complete idea. To better understand
-               this, consider a hard copy book which has sections. Each section belongs to the book.
-               There could be many books in a library. In this example, the "library" is the entire
-               Web site.
+            <p>目次とナビゲーションバーやサイトマップのような他のナビゲーション要素との間には違いがある。目次は同じドキュメントのセクションへのリンクを提供する。それらのセクションは、同じウェブページに配置する、又は複数のウェブページにまたがって配置することができる。しかしいずれでも構想を完成させることができる。このことをさらに理解するために、複数のセクションのあるハードコピーの本を考えてみよう。各セクションは本に属する。図書館には多くの本がある。この例では「図書館」が全体のウェブサイトに相当する。
```

Comment:
最初の文の構造がわかりにくいので、例えば以下のような訳ではいかがでしょうか？

* 読点を追加
```suggestion
            <p>目次と、ナビゲーションバーやサイトマップのような他のナビゲーション要素との間には違いがある。目次は同じドキュメントのセクションへのリンクを提供する。それらのセクションは、同じウェブページに配置する、又は複数のウェブページにまたがって配置することができる。しかしいずれでも構想を完成させることができる。このことをさらに理解するために、複数のセクションのあるハードコピーの本を考えてみよう。各セクションは本に属する。図書館には多くの本がある。この例では「図書館」が全体のウェブサイトに相当する。
```
* between A and B の構造をくずしたもの
```suggestion
            <p>目次は、ナビゲーションバーやサイトマップのような他のナビゲーション要素とは異なる。目次は同じドキュメントのセクションへのリンクを提供する。それらのセクションは、同じウェブページに配置する、又は複数のウェブページにまたがって配置することができる。しかしいずれでも構想を完成させることができる。このことをさらに理解するために、複数のセクションのあるハードコピーの本を考えてみよう。各セクションは本に属する。図書館には多くの本がある。この例では「図書館」が全体のウェブサイトに相当する。
```